xc/xenops_server_xen.ml

```diff
@@ -1903,6 +1903,7 @@ module VGPU = struct
 				let emulator_pid =
 					match vgpu.implementation with
 					| GVT_g _ -> Device.Qemu.pid ~xs frontend_domid
+					| MxGPU _
```

Comment:
I've updated this now so that for MxGPU we use **`Qemu`**`.pid`, not **`Vgpu`**`.pid`.